glommio/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "glommio"
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
`examples/Cargo.toml` `[dependencies] glommio = ...` should be updated to match the new version number.

Reply:
you are faster than the CI =p